README.md

```diff
@@ -37,52 +39,46 @@ spring.data.mongodb.uri=${MONGODB_ADDON_URI}
 
 Where `MONGODB_ADDON_URI` is an environment variable which is specified by MongoDB addon link.
 
-### Clever Cloud
+## Install and run locally
 
-For a maven application on clever, you have to define `clevercloud/maven.json` which is containing the maven goals to build and run your app.
-```json
-{
-    "build": {
-        "goal": "package"
-    },
-    "deploy": {
-        "goal": "spring-boot:run"
-    }
-}
-```
+-   install dependencies: `mvn install`
+-   tests: `mvn test`
+-   run: `mvn spring-boot:run`
 
-## Install on Clever Cloud
+## Deploy on the Clever Cloud console
 
-Create a clever application using maven, install a MongoDB addon, link it then deploy !
+Create a mongodb add-on and link it to your app.
 
-## Install and run locally
+### Set the goal in the environment
 
-* install dependencies: `mvn install`
-* tests: `mvn test`
-* run: `mvn spring-boot:run`
+In the environment variables, create `MAVEN_DEPLOY_GOAL` and set it to `spring-boot:run`
 
 ## Example
 
 Create new user
+
 ```bash
-curl -d '{"firstName":"DUVAL", "lastName":"Alexandre"}' -H "Content-Type: application/json" -X POST http://<appId>.cleverapps.io/users
+curl -d '{"firstName":"DUVAL", "lastName":"Alexandre"}' -H "Content-Type: application/json" -X POST https://<appId>.cleverapps.io/users
```

Comment:
Robert Cloud?

Reply:
it was here before I swear :-D

Reply:
I mean, can you apply it here too. Then auto-merge it if you need it for the demo tomorrow otherwise I'll do it.